libraries/achartengine/achartengine/src/org/achartengine/GraphicalActivity.java

```diff
@@ -15,34 +15,59 @@
  */
 package org.achartengine;
 
-import org.achartengine.chart.AbstractChart;
-
+import android.app.ActionBar;
 import android.app.Activity;
+import android.os.Build;
 import android.os.Bundle;
+import android.view.MenuItem;
 import android.view.Window;
 
+import org.achartengine.chart.AbstractChart;
+
 /**
  * An activity that encapsulates a graphical view of the chart.
  */
 public class GraphicalActivity extends Activity {

```

Comment:
Is there a reason this doesn't just extend CommCareActivity? Would make the back behavior and such much easier to encapsulate.